src/orquestra/sdk/_base/_driver/_ce_runtime.py

```diff
@@ -525,17 +535,58 @@ def get_workflow_logs(self, wf_run_id: WorkflowRunId) -> WorkflowLogs:
         for sys_m in sys_messages:
             system_logs.add_line_by_stream(LogStreamType.STDOUT, str(sys_m.log))
 
+        # If we don't have any logs and we have ray worker logs, we should use these
+        # as the "unknown" task invocation logs. This is to ensure backwards compat
```

Comment:
**nitpick**: `"UNKNOWN TASK INV ID"` was a temporary placeholder I'd prefer to get rid of as soon as we support task log correlation. We already have `.other` logs as a "plan B" for unforseen situations. Is there a way to use `.other` in this scenario?

Reply:
Should we always put in these worker logs? they're basically empty after we intercept stdout/stderr

Reply:
> Should we always put in these worker logs

Not sure if I understand correctly. If you mean taking worker logs from the API response and exposing this data under `WorkflowLogs.other` then yeah, sounds good. Otherwise, please elaborate :)

Reply:
What I meant was should we always do this or should we only add the worker logs when we didn't have any task run logs?